test/services/return-requirements/delete-note.service.test.js

```diff
@@ -0,0 +1,64 @@
+// write test in hapi/yar for delete-note.service.js
```

Comment:
I'm guessing this was a note to yourself!

So, we need to drop it. Also, we always add `'use strict'` to the top of all our files.

```suggestion
'use strict'

```